apps/examples/testcase/le_tc/kernel/tc_libc_stdlib.c

```diff
@@ -46,24 +46,35 @@
 #define BSEARCH_ARRSIZE 10
 
 /**
-* @fn                   :compare
-* @description          :Function for tc_libc_stdlib_qsort
-* @return               :int
-*/
+ *@fn					:thread_func
+ *@description			:Function for tc_libc_stdlib_abort
+ *@return				:void*
+ */
+static void *thread_func(void *arg)
+{
+	abort();
```

Comment:
As the implementation of abort function internally calls pthread_exit / exit, it would only stop the calling thread not the parent process. That's why i created a thread called abort in that thread.